libzmix/bbs/src/verifier.rs

```diff
@@ -71,4 +71,35 @@ impl Verifier {
     pub fn generate_proof_nonce() -> ProofNonce {
         ProofNonce::random()
     }
+
+    /// create the challenge hash for a set of proofs
+    ///
+    /// # Arguments
+    /// * `proofs` - a vec of SignatureProof objects
+    /// * `proof_requests` - a corresponding vec of ProofRequest objects
+    /// * `nonce` - a SignatureNonce
+    /// * `claims` - a vec of strings the prover wishes to include in the challenge (may be empty)
+    pub fn create_challenge_hash(
+        proofs: &[SignatureProof],
+        proof_requests: &[ProofRequest],
+        claims: &[&[u8]],
```

Comment:
Maybe this should be optional. I'd like to use the method but I don't necessarily have any additional claims to add.

`claims: Option<&[&[u8]]>`